include/picongpu/simulation_defines/param/laser.param

```diff
@@ -383,4 +383,93 @@ namespace picongpu
         }
     }
 
+#ifndef PARAM_INT_3
+#define PARAM_INT_3 1.e-5
```

Comment:
please do not add those pre-compiler macros, they are not needed (and also ill-named).

You can still add them in your local example if you need to control them from e.g. `cmakeFlags`, but they should not be part of the PR.